okhttp/src/main/java/okhttp3/internal/cache/CacheStrategy.kt

```diff
@@ -167,21 +167,21 @@ class CacheStrategy internal constructor(
       val ageMillis = cacheResponseAge()
       var freshMillis = computeFreshnessLifetime()
 
-      if (requestCaching.maxAgeSeconds() != -1) {
-        freshMillis = minOf(freshMillis, SECONDS.toMillis(requestCaching.maxAgeSeconds().toLong()))
+      if (requestCaching.maxAgeSeconds != -1) {
+        freshMillis = minOf(freshMillis, SECONDS.toMillis(requestCaching.maxAgeSeconds.toLong()))
```

Comment:
I suggest an extension function `TimeDuration.toMillis(Int)` in UtilKt to avoid the toLong operations here.